substantial/workflows/context.py

```diff
@@ -144,3 +148,23 @@ def cancel_run(self):
         Interrupt after the call
         """
         raise CancelWorkflow(self.run)
+
+
+class Utils:
+    def __init__(self, ctx: Context):
+        self.__ctx = ctx
+
+    async def now(self, tz: Optional[timezone] = None) -> datetime:
+        now = await self.__ctx.save(lambda: datetime.now(tz).isoformat())
+        return datetime.fromisoformat(now)
```

Comment:
just by adding a variable, we could save this line, this is definitely worth to do

Reply:
https://github.com/zifeo/substantial/pull/10#discussion_r1838261848

Reply:
you can still save the value and avoid the parsing